content/docs/2.12/operate/opentelemetry.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Integrate with OpenTelemetry Collector"
+date: 2023-08-30T11:13:57+08:00
+draft: true
+---
+
+## Push Metrics to OpenTelemetry Collector
+
+### Operator
+
+The KEDA Operator supports outputting metrics to the OpenTelemetry collector using HTTP. The parameter "--enable-opentelemetry-metrics=true" needs to be set. KEDA will push metrics to the OpenTelemetry collector specified by the "OTEL_EXPORTER_OTLP_ENDPOINT" environment variable. Other environment variables in OpenTelemetry are also supported (https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/). Here is an customize YAML example of Operator: 
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: keda-operator
+  ...
+      containers:
+        - name: keda-operator
+          image: ghcr.io/kedacore/keda:latest
+          command:
+            - /keda
+          args:
+            --enable-opentelemetry-metrics=true
+            ...
+          ...
+          env:
+            - name: OTEL_EXPORTER_OTLP_ENDPOINT
```

Comment:
We should be future proof and add the protocol in the name so that if we decide to add gRPC in the future it still works.

Unless we move the protocol to another field of course :)